packages/genesys-spark-components/src/components/beta/gux-time/tests/__snapshots__/gux-time.spec.ts.snap

```diff
@@ -6875,7 +6875,7 @@ exports[`gux-time #render using regional dates #render different time zones shou
 exports[`gux-time #render using regional dates #render different time zones should work as expected for "Antarctica/Casey" 1`] = `
 <gux-time-beta datetime="2022-07-07T12:00:00.000Z" format="full" time-zone="Antarctica/Casey">
   <mock:shadow-root>
-    11:00:00 PM Casey Time
+    8:00:00 PM Australian Western Standard Time
```

Comment:
https://timezonedb.com/time-zones/Antarctica/Casey
After Thursday, 09 March, 2023 02:59:59 AM: Clocks were moved backward to become Thursday, 09 March, 2023 12:00:00 AM